back/engines/commercial/public_api/config/routes.rb

```diff
@@ -19,6 +19,7 @@
 
     resources :files, concerns: :deleted_items
     resources :ideas, only: %i[index show create update], concerns: :deleted_items
+    resources :internal_comments, only: %i[index show create], concerns: :deleted_items
```

Comment:
Why are we adding internal comments to the public API? Is this related to idea assignment somehow?